tf_seal/cc/ops/seal_ops.cc

```diff
@@ -72,3 +72,18 @@ REGISTER_OP("SealPolyEval")
     .Input("pub_key: variant")
     .Output("out: variant")
     .SetIsStateful();
+
+REGISTER_OP("SealSavePublickey")
+    .Input("pub_key: variant")
+    .Output("out:variant")
+    .SetIsStateful();
+
+REGISTER_OP("SaveSecretkey")
```

Comment:
`SealSaveSecretkey` (to match `SealSavePublickey`)?

Reply:
I want to save both, public and secret. I saw a small typo on kernel

Reply:
oh, I mean that you've left out the `Seal` prefix on this one

Reply:
Done